addons/sale/views/sale_order_views.xml

```diff
@@ -282,7 +282,11 @@
                 </div>
                 <group name="sale_header">
                     <group name="partner_details">
-                        <field name="partner_id" widget="res_partner_many2one" context="{'res_partner_search_mode': 'customer', 'show_address': 1, 'show_vat': True}" options='{"always_reload": True}' placeholder='Type to find a customer...'/>
+                        <field name="partner_id" 
+                               widget="res_partner_many2one" 
+                               context="{'res_partner_search_mode': 'customer', 'show_address': 1, 'show_vat': True}" 
+                               options='{"always_reload": True}' 
```

Comment:
```suggestion
                               options="{'always_reload': True}"
```
Pure nitpicking, but that way the definition of all parameters will be the same ^^